### ISSUE TITLE
CIFAR10 input preprocessing

### ISSUE THREAD

**Issue by honglh** [jiecaoyu/XNOR-Net-PyTorch#86]
Hi, can you share the preprocessing you have done to the CIFAR10 training set that results in the "train_data" in your module input? I used normalization on each channel but results does not match your dataset

Thanks!

**Comment by jiecaoyu** [comment]
@honglh Hi, please check this [link](https://gist.github.com/mavenlin/e56253735ef32c3c296d) for details. Basically, the processing steps are following the Network in Network paper.

**Comment by honglh** [comment]
Thanks! appreciate your work
